### ISSUE TITLE
 dyld[3986]: Library not loaded: @rpath/libclang.dylib

### ISSUE THREAD

**Issue by kushwahashiv** [shekohex/flutterust#35]
<img width="1638" alt="image" src="https://user-images.githubusercontent.com/4386810/158437076-23bc2ffe-cdbe-4b0d-b997-eb68b1f567c2.png">

I am still getting the issue even I followed the readme instructions

**Comment by shekohex** [comment]
Have you installed `llvm`? if you are on macOS `brew install llvm`

**Comment by kushwahashiv** [comment]
yes I did

**Comment by cameroncooper** [comment]
I got past this by setting `DYLD_FALLBACK_LIBRARY_PATH` to the location containing `libclang.dylib`. 

For example `export DYLD_FALLBACK_LIBRARY_PATH=/Library/Developer/CommandLineTools/usr/lib`

**Comment by tGrothmannFluffy** [comment]
I had the same issue.
After following @cameroncooper's advice to set `DYLD_FALLBACK_LIBRARY_PATH` I am now getting the following error:

```
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo:
archive member /Users/fluffy/git/flutterust/target/x86_64-apple-ios/debug/libscrap_ffi.a(aesni-x86_64-macosx.o) 
cputype (16777228) and cpusubtype (0) does not match previous archive members cputype (16777223) and cpusubtype (3) (all members must match)

[ERROR cargo_lipo] Executing "lipo" "-create" "-output" "/Users/user/git/flutterust/target/universal/debug/libscrap_ffi.a" "/Users/user/git/flutterust/target/aarch64-apple-ios/debug/libscrap_ffi.a" "/Users/user/git/flutterust/target/x86_64-apple-ios/debug/libscrap_ffi.a" finished with error status: exit status: 1
```

I'm assuming this could have something to do with running on an M1...

**Comment by shekohex** [comment]
> I had the same issue. After following @cameroncooper's advice to set `DYLD_FALLBACK_LIBRARY_PATH` I am now getting the following error:
> 
> ```
> fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo:
> archive member /Users/fluffy/git/flutterust/target/x86_64-apple-ios/debug/libscrap_ffi.a(aesni-x86_64-macosx.o) 
> cputype (16777228) and cpusubtype (0) does not match previous archive members cputype (16777223) and cpusubtype (3) (all members must match)
> 
> [ERROR cargo_lipo] Executing "lipo" "-create" "-output" "/Users/user/git/flutterust/target/universal/debug/libscrap_ffi.a" "/Users/user/git/flutterust/target/aarch64-apple-ios/debug/libscrap_ffi.a" "/Users/user/git/flutterust/target/x86_64-apple-ios/debug/libscrap_ffi.a" finished with error status: exit status: 1
> ```
> 
> I'm assuming this could have something to do with running on an M1...

Yeah, maybe, I have not test it on M1 machine before, but you will need to build the lib for aarch64 or arm basically, instead of x86_64

**Comment by tGrothmannFluffy** [comment]
🤔 Could you point me to where and what I would have to change?
